src/parsers/power_system_table_data.jl

```diff
@@ -1047,8 +1052,9 @@ function make_thermal_generator_multistart(
         no_load_cost = 0.0
     else
         (no_load_x, no_load_cost) = first(var_cost)
+        @warn "Strange math occurring here (part 1)"
```

Comment:
@jd-lara, could you check what we're doing with `no_load_cost` here and at `part 2` below? I don't entirely understand why we are subtracting this out in the first place, and I especially don't get why we're subtracting something from power as well as from cost. `part 1` here is a [direct translation of the old logic](https://github.com/NREL-Sienna/PowerSystems.jl/pull/1056/files#diff-25b200340315b2133dda94fd2e46f429b1a28a3ae9c9e63e26813d3c5865e7cbR1052); `part 2` is adding `no_load_cost` to `fixed` because we don't have a `no_load_cost` field in `ThermalGenerationCost`. I think this code is not covered by tests, the test [here](https://github.com/NREL-Sienna/PowerSystems.jl/pull/1071/files#diff-f7ca8d41191dcc468819e4637c4b7e9c23b36ff33054a881572f6d4d8c59cddaR105) doesn't test any `ThermalMultiStart`s.